pyscf_ipu/experimental/integrals.py

```diff
@@ -24,13 +24,22 @@
 """
 
 
-def factorial(n: IntN, nmax: int = LMAX) -> IntN:
+def factorial_fori(n: IntN, nmax: int = LMAX) -> IntN:
     def body_fun(i, val):
         return val * jnp.where(i <= n, i, 1)
 
     return lax.fori_loop(1, nmax + 1, body_fun, jnp.ones_like(n))
 
 
+def factorial_gamma(n: IntN) -> IntN:
+    from jax.scipy.special import gammaln
```

Comment:
I would think it's OK to import this at the top of the file.

Reply:
Or maybe this file is not intended for this PR?

Reply:
Oh yes, I see from the tests that it is.

Reply:
This was already imported at the top of the file 🤦